Feature/96 implement visual novel search

NightOfNights opened the pull request:
Didn't implement manipulation with UrlSearchParams and filter novel tags. I think I will move it out as a separate task and implement later
close #96

NightOfNights committed:
chore: update eslint config #96

Moltenship committed:
feat: add base staff page #80

Moltenship committed:
feat: add base table #80

moetazaneta committed:
fix: add proper path to overview staff links #80

Moltenship committed:
refactor: update gender model #80

Moltenship committed:
fix: update docs #80

Moltenship committed:
fix: remove typecasting #80

NightOfNights committed:
Merge branch 'develop' into feature/96-implement-visual-novel-search

Moltenship committed:
fix: simplify template, move out interface to model #80

Moltenship committed:
fix: remove `useMemo` #80

Moltenship committed:
fix: update docs #80

NightOfNights committed:
fix: improve grammar of some docs #96

NightOfNights committed:
Merge remote-tracking branch 'origin/feature/97-search-cards' into feature/96-implement-visual-novel-search

NightOfNights committed:
fix: repair chinese language icon #96

NightOfNights committed:
fix: add missing keys #96

NightOfNights committed:
feat: implement text input component #96

NightOfNights committed:
feat: add modal with search filters #96

Moltenship committed:
feat: add date sorting #80

Moltenship committed:
fix: rename compare function #80

Moltenship committed:
fix: update docs #80

NightOfNights committed:
feat: implement visual novel searching #96

NightOfNights committed:
fix: remove unused import #96

NightOfNights committed:
chore: improve memoization #96

moetazaneta reviewed (approved):
Great work!

moetazaneta committed:
feat: change button sizes #96

Moltenship committed:
Merge pull request #117 from vndbre/feature/80-staff-page

Feature/80 staff page

NightOfNights committed:
refactor: improve namings #96

NightOfNights committed:
fix: add missing import #96

NightOfNights committed:
Merge remote-tracking branch 'origin/feature/97-search-cards' into feature/96-implement-visual-novel-search

NightOfNights committed:
Merge branch 'develop' into feature/96-implement-visual-novel-search

Moltenship commented on src/features/Search/components/VisualNovelSearchForm/VisualNovelSearchForm.tsx:
```diff
@@ -1,153 +1,194 @@
+                />
+
+                <MultiSelect
+                  control={control}
+                  name="platforms"
+                  label="Platform"
+                  options={platformOptions}
+                  placeholder="Any"
+                  displayLimit={1}
+                  closeMenuOnSelect={false}
```
You can use object for default `multiselect` options  like `placeholder`, `displayLimit` and others to reduce code duplication

prekel committed:
chore: netlify rewrite

prekel committed:
chore: netlify badge

prekel committed:
Merge pull request #125 from vndbre/netlify

Netlify

dependabot[bot] committed:
chore(deps): bump minimist from 1.2.5 to 1.2.6

Bumps [minimist](https://github.com/substack/minimist) from 1.2.5 to 1.2.6.
- [Release notes](https://github.com/substack/minimist/releases)
- [Commits](https://github.com/substack/minimist/compare/1.2.5...1.2.6)

---
updated-dependencies:
- dependency-name: minimist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

prekel committed:
Merge pull request #126 from vndbre/dependabot/npm_and_yarn/minimist-1.2.6

chore(deps): bump minimist from 1.2.5 to 1.2.6

prekel committed:
Delete azure-static-web-apps-witty-pebble-0b58b0803.yml

moetazaneta committed:
Merge pull request #120 from vndbre/feature/97-search-cards

Feature/97 Search page cards

prekel committed:
Merge pull request #128 from vndbre/remove-azure-static-web-app

Delete azure-static-web-apps-witty-pebble-0b58b0803.yml

NightOfNights committed:
Merge branch 'develop' into feature/96-implement-visual-novel-search

NightOfNights committed:
Merge branch 'develop' into feature/96-implement-visual-novel-search
